scripts/bootstrap-prefix.sh

```diff
@@ -2221,11 +2286,37 @@ bootstrap_stage3() {
 		ln -s bash "${ROOT}"/bin/sh
 	fi
 
-	# Start using apps from the final destination Prefix
-	export PREROOTPATH="${ROOT}/usr/bin:${ROOT}/bin"
+	# Start using apps from new target
+	cat > "${ROOT}"/tmp/etc/env.d/10stage3 <<-EOF
+		PATH="${ROOT}/usr/bin:${ROOT}/bin:${ROOT}"
```

Comment:
what is the trailing ROOT supposed to be here?  what lives there?

Reply:
Not sure, maybe it was a typo.

I no longer have Mac available, so can't test what happens without it.